domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -1252,16 +1313,17 @@ public void saveBatchEmpty(TestContext context) {
     List<Object> list = Collections.emptyList();
     createFoo(context).saveBatch(FOO, list, res -> {
       assertSuccess(context, res);
-      context.assertEquals(0, res.result().getRows().size());
-      context.assertEquals("id", res.result().getColumnNames().get(0));
+      context.assertEquals(null, res.result());
```

Comment:
This is an unneccessary breaking change that is inconsistent with vertx-pg-client that returns an empty RowSet when "UPDATE ... WHERE ..." does not update any record.
Can you return an empty RowSet?

Reply:
Where the first column name is `id`?

Reply:
Yes